src/main/java/me/karakelley/tictactoe/GameConfiguration.java

```diff
@@ -41,13 +43,20 @@ public Player getPlayer1() {
     return player1;
   }
 
-  public void chooseOpponent(String choice) {
-    if (!choice.equals(HUMAN_PLAYER_SELECTION)) System.exit(0);
-    player2 = playerFactory.makePlayer(HUMAN_PLAYER_SELECTION, getPlayer2marker());
+  private void configureUIMessages() {
+    if (gameType.equals(EASY_PLAYER_SELECTION)) {
```

Comment:
OCP?

Reply:
I thought about making different interfaces and then using a factory to create the interface needed based on the game type but I felt that it might be more complex than necessary since I only need to change two values. Thoughts on that approach?